is_core/auth_token/middleware.py

```diff
@@ -33,10 +33,12 @@ def process_response(self, request, response):
         # Save the session data and refresh the client cookie.
         # Skip session save for 500 responses, refs #3881.
         if response.status_code != 500 and hasattr(request, 'token') and request.token.is_active:
-            if not request.token.expiration:
```

Comment:
no je to trošku naruby:

když má nastavenu expiraci, znamená to, že nezaškrtl "trvale přihlášen" to znamená, že se cookie musí smazat po uzavření prohlížeče, pokud na cookie nastavíš expiraci tak se nesmaže

když nemá nastavenu expiraci tak se používá max doba expirace, která je v settings is-core jelikož nějaká vždy musí existovat (to je dva týdny), na cookie se tato expirace nastaví a to zůstane funkční i po uzavření prohlížeče. Je to jasné?